.devcontainer/devcontainer.json

```diff
@@ -15,7 +15,7 @@
     },
     "postCreateCommand": "/bin/bash .devcontainer/postCreate.sh ${containerWorkspaceFolder}",
     "postAttachCommand": {
-        "rstudio-start": "sudo rstudio-server start"
+        "rstudio-start": "sudo rstudio-server start && /opt/venv/bin/activate"
```

Comment:
Why is this change needed as part of adding the read permission for Stata? This feels like it's part of a separate change?

Reply:
Huh. It is. That's very weird. It is part of a PR that was merged earlier this morning. I think a rebase is in order. One moment please!

Reply:
Wait, no it isn't. It's something left over from an earlier change that ended up being implemented in the docker file. I'm a wally, good catch!